javav2/example_code/ecs/src/test/java/EcsTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+import com.example.ecs.*;
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.ecs.EcsClient;
+import java.io.*;
+import java.util.*;
+
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class EcsTest {
+
+    private static  EcsClient ecsClient;
+    private static Region region;
+    private static String clusterName = "";
+    private static String clusterARN = "";
+    private static String taskId = "";
+    private static String securityGroups = "";
+    private static String subnet = "";
+    private static String serviceName = "";
+    private static String serviceArn = "";
+    private static String taskDefinition = "";
+
+    @BeforeAll
+    public static void setUp() throws IOException {
+
+        // Run tests on Real AWS Resources
+        region = Region.US_EAST_1;
+        ecsClient = EcsClient.builder()
+                .region(region)
+                .build();
+
+
+        try (InputStream input = EcsTest.class.getClassLoader().getResourceAsStream("config.properties")) {
+
+            Properties prop = new Properties();
+
+            if (input == null) {
+                System.out.println("Sorry, unable to find config.properties");
+                return;
+            }
+
+            //load a properties file from class path, inside static method
```

Comment:
load -> Load

Also a period at the end of the sentence.

Reply:
fixed